packages/gatsby-admin/src/pages/index.tsx

```diff
@@ -1,19 +1,30 @@
 /** @jsx jsx */
-import React from "react"
-import { jsx, Flex, Grid } from "strict-ui"
+import React, { Fragment } from "react"
+import { jsx, Flex } from "strict-ui"
 import { Spinner } from "theme-ui"
 import { useQuery, useMutation } from "urql"
 import {
+  Card,
   Heading,
-  Text,
   DropdownMenu,
   DropdownMenuButton,
   DropdownMenuItem,
   DropdownMenuItems,
+  HeadingProps,
+  Text,
+  Badge,
+  Spacer,
 } from "gatsby-interface"
+// TODO: teach TypeScript about SVGs
```

Comment:
This is as simple as adding a d.ts file somewhere in the root with `declare module "*.svg"`